recipe/meta.yaml

```diff
@@ -11,7 +11,6 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  skip: true  # [win]
```

Comment:
> Or are msvcrt14 and msvcrt9 not compatible in the same process? Then I believe I could statically link to msvcrt.

Correct they are not compatible.

> Or maybe it is just not worth the pain, and I'll see what to do when Py2 on Windows users will appear.

I'd try just Python 3.5 for now and see how that goes.

A long time ago I suggested that we build a non-standard Python 2.7 against VS2015. However, it was shot down even by people who do want to use C++11 with Python 2.7 on Windows. That being said, I think it would be possible to do so outside conda-forge and know people who do this.